x-pack/plugins/cases/public/components/all_cases/assignees_filter.test.tsx

```diff
@@ -309,4 +311,28 @@ describe('AssigneesFilterPopover', () => {
     fireEvent.change(screen.getByPlaceholderText('Search users'), { target: { value: 'dingo' } });
     expect(screen.queryByText('No assignees')).not.toBeInTheDocument();
   });
+
+  it('shows warning message when reaching maximum limit to filter', async () => {
+    const maxAssignees = Array(MAX_ASSIGNEES_FILTER_LENGTH).fill(userProfiles[0]);
+    const props = {
+      ...defaultProps,
+      selectedAssignees: maxAssignees,
+    };
+    appMockRender.render(<AssigneesFilterPopover {...props} />);
+
+    await waitFor(async () => {
+      userEvent.click(screen.getByTestId('options-filter-popover-button-assignees'));
```

Comment:
_nit_: we use the old  `"@testing-library/user-event": "^13.5.0",` which doesn't return a promise([I think it is here](https://github.com/testing-library/user-event/blob/v13.5.0/src/click.ts#L117)) so this could be moved outside.

Reply:
I tried to move `userEvent.click(screen.getByTestId('options-filter-popover-button-assignees'));` out of `waitFor` but test fails due to it.

Reply:
Weird, [in the documentation for this version they don't use it.](https://testing-library.com/docs/ecosystem-user-event/#clickelement-eventinit-options) I don't know what to believe anymore 😆

Reply:
FWIW, we can convert it to `userEvent.click(await screen.findByTestId('options-filter-popover-button-assignees'));` and remove the `waitFor`. All flavors of `find*` await the element.